packages/fuel-indexer-tests/contracts/fuel-indexer-test/src/main.sw

```diff
@@ -15,6 +15,12 @@ use std::{
     token::*,
 };
 
+configurable {
```

Comment:
@segfault-magnet Here's the usage of `configurables` that SwayGang had (the link that was giving you a 404)